unison-src/transcripts/error-messages.output.md

```diff
@@ -195,6 +208,56 @@ foo = cases
         4 |   3 -> ()
     
 
+```
+```unison
+-- Missing a '->'
+x = match Some a with
+      None -> 
+        1
+      Some _
+        2
+```
+
+```ucm
+
+  offset=16:
```

Comment:
Anyone know what these `offsets` are in our parser error messages? Doesn't seem helpful to me 🤔